packages/e2e-test-utils-playwright/src/request-utils/comments.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Internal dependencies
+ */
+import type { RequestUtils } from './index';
+
+export interface Comment {
+	id: number;
+	author: number;
+	content: string;
+}
+
+/**
+ * Create new comment using the REST API.
+ *
+ * @param {} this    RequestUtils.
+ * @param {} comment Comment.
+ */
+export async function createComment( this: RequestUtils, comment: Comment ) {
+	this.rest( {
```

Comment:
Missing `await` here?

We should probably return the response of the API too to capture the created comment id.